### DISCUSSION TITLE
How to build

### DISCUSSION THREAD

**Discussion by selescop** [ARMmbed/cmsis-pack-manager#231] [General]
Hello,
Sorry for the dummy question but how do you build the python package?
I read from the doc:

> run python2 setup.py bdist_wheel from the root of this repo

but there is no longer 'setup.py' (and I guess you went to python3 now).
Thank you

**Comment by mbrossard** [comment]
CI uses maturin directly:
```
$ pip3 install maturin cffi
$ maturin build --release
```

You can also use a build front-end like `build`:
```
$ pip3 install build
$ python3 -m build
```
